### PR TITLE
Russian translation of "Breaking Monero ep. 7-13" and "MoneroKon transcription"

### PR DESCRIPTION
**Pull request by v1docq47**
_add:_
>   monero-outreach-docs/translations/ru/transcriptions/monero-konferenco-2019/

_translated:_
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/07_remote_nodes_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/08_timing_attacks_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/09_poisoned_outputs_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/10_public_mining_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/12_in_output_metadata_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/breaking_monero/13_blockchain_explorer_ru.md

>   monero-outreach-docs/translations/ru/transcriptions/monero-konferenco-2019/ASIC-Resistant_Proof_of_Work_Fact_or_Fantasy_ru.md
>   monero-outreach-docs/translations/ru/transcriptions/monero-konferenco-2019/The_Embedded_Future_of_Monero's_PoW_ru.md

Calling to review:
@monerorus @LvMsterfild @TheFuzzStone

### CODE REVIEW

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+[Monero Konferenco](https://monerokon.com/)  
+
+[Все эпизоды Breaking Monero](https://www.monerooutreach.org/breaking-monero/)
+
+_**Стенограмма эпизода:**_
+
+_**Джастин:**_ Добро пожаловать на одиннадцатый эпизод Breaking Monero. Это специальный эпизод, в рамках которого мы ответим на вопросы, присланные нам на почту людьми в связи с прошлыми эпизодами. Мы — это я, Джастин, Брендон и Саранг. На самом деле, вопросов довольно много, и мы надеемся, что у нас получится восполнить некоторые пробелы или разъяснить некоторые серии Breaking Monero, или же коснуться проблем, которые, по вашему мнению, имеют отношение к контрибьюторам Исследовательской лаборатории Monero. Итак, Брендон, Саранг, начнём, пожалуй.
+
+_**Джастин:**_ В первом полученном нами вопросе, касающимся темы, затронутой во втором эпизоде, говорится о том, что во время одного из эпизодов Monero Talk Эндрю Поэлстра высказал идею, согласно которой Monero может либо обеспечивать совершенную анонимность, используя свои механизмы сокрытия средств, либо твёрдую гарантию невозможности создания таких средств. Не мог бы ты сравнить идеальное связывание с идеальным сокрытием, Саранг?
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: Hey @v1docq47,

Will leave you comments on the paragraphs I've seen that  are missing links. This one is one of them, you will just need to add them >:).

- [episode 2](https://www.monerooutreach.org/breaking-monero/ring-signatures.php)
- [Monero Talk](https://www.monerotalk.live/)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
...ом она также всё переворачивает, так что если проект и решит использовать схему обязательств Эль-Гамаля, это, по сути, будет означать, что сокрытие суммы более не будет идеальным, а значит, злоумышленник, не ограниченный в вычислительных ресурсах, сможет узнать скрытую сумму, но в то же время вы будете идеально связаны, то есть не сможете позднее открыть это обязательство для другой суммы. Идея концепции, к которой мог бы перейти или не перейти проект Monero, которая бы изменила ситуацию, довольно сложна. Такое решение должно соответствовать типам используемых нами подписей, а также используемому нами виду доказательств диапазона. В работе по Bulletproofs, например, рассматривалась возможность изменения протокола таким образом, чтобы он бы стал более сложным с вычислительной точки зрения и в количестве требуемого места, что могло бы подтолкнуть к переходу к обязательствам Эль-Гамаля. Но, опять же, это должно было бы соответствовать другим используемым нами схемам. В сущности, этот вопрос особо не рассматривается, потому как мы накладываем определённые вычислительные ограничения на злоумышленников и пользователей системы. Брендон, может, ты что-то добавишь по теме схем обязательств?
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [Bulletproofs paper](https://eprint.iacr.org/2017/1066.pdf)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
...о издание даёт хорошее представление о некоторых классических и современных методах шифрования. Предполагается, что для начала у вас уже имеется базовое знание алгебры уровня средней школы или колледжа. Это хорошая книга для начала. То есть с точки зрения продвинутых методов она не так хороша, равно как и для создания безопасных решений. В ней рассказывается о математике, стоящей за такими решениями, но я бы не полагался на её содержание при создании безопасного варианта реализации. Тем не менее это очень хорошая книга. Следующая книга пригодится тем, кто больше заинтересован в формальном подходе, и она не для слабонервных. Это Введение в современную криптографию Каца и Линделла. Эта книга предназначена для продвинутых студентов и начинающих аспирантов. В ней описаны формальные варианты реализации определённых конструкций. Это очень-очень хорошая и очень-очень полная книга. Я бы не советовал браться за неё новичкам. В ней очень хорошо описаны формальные варианты. Есть ещё множество хороших книг, но эти две охватывают два важных уровня. Они будут полезны тем, кто начинает разбираться с математикой и действительно хочет усилить доказательства безопасности и заняться подобными вещами.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [Introduction to Mathematical Cryptography](https://www.goodreads.com/book/show/3884171-an-introduction-to-mathematical-cryptography)
- [Introduction to Modern Cryptography](https://www.goodreads.com/book/show/24387035-introduction-to-modern-cryptography-second-edition)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+_**Шурэ:**_ Я только на днях узнал о существовании документа под названием Интенсивное введение в криптографию, который представляет собой набор слайдов, и который был написан в Гарварде. В настоящее время документ используется в Университете Клемсона в рамках курса по криптографии.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [Intensive Introduction to Cryptography](https://intensecrypto.org/public/index.html)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+_**Джастин:**_ И что в нём хорошо, так это то, что он выложен в свободный доступ в формате PDF, так что вы совершенно спокойно можете ознакомиться с ним.
+
+_**Шурэ:**_ Всегда приятно скачивать бесплатные PDF. Я не знаю другого способа бесплатного получения PDF, кроме как напрямую у самих авторов.
+
+_**Джастин:**_ Итак, я думаю с этим достаточно, давайте переходить к следующему вопросу. Он связан с нашим пятым эпизодом и касается того, как люди будут использовать искусственный интеллект и другие технологии для усиления эвристического анализа блокчейна Monero. Мы установили, что нет предела уровню эвристического анализа, к которому могут прибегнуть люди, чтобы получить информацию, касающуюся транзакций Monero, и у зрителей возник вопрос, основан ли выбор выходов Monero на факторах, связанных с реальным выходом, или же ложные выходы выбираются совершенно независимо от них?
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [fifth episode](https://www.monerooutreach.org/breaking-monero/input-selection-algorithm.php)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+_**Джастин:**_ Спасибо вам обоим. Мы перейдём к другим вопросам, а к моделям угроз вернёмся позже. Сейчас я собираюсь рассмотреть вопрос, связанный с седьмым эпизодом и требующий более подробного разъяснения разницы между анонимностью, обеспечиваемой Monero, и анонимностью в Lightening Network. Также нас просят подробнее рассказать о том, как реализация Lightening Network и других «оффчейн» решений может сказаться на Monero? Саранг, не мог бы ты немного поделиться, какое влияние оказывают эти оффчейн решения?
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [seventh episode](https://www.monerooutreach.org/breaking-monero/remote-nodes.php)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+_**Джастин:**_ Хорошо, ещё раз спасибо. Следующий вопрос касается нашего девятого эпизода. Многие наблюдали, как мы втроём обсуждали и тестировали эвристический подход, как мы делились эвристическим подходом, применимым к блокчейну Monero, и зрители были очень разочарованы, они начали говорить: «Ух ты! Оказывается, есть так много способов получить информацию о том, как люди пользуются Monero». Будет ли когда-нибудь найдено решение, которое позволит нам совершенно забыть об этой проблеме, появится ли тот чёрный ящик, который скроет от всех происходящее внутри него? Так как разговор идёт о компромиссах, начнём с тебя, Шурэ. Что бы ты ответил людям на их заявления вроде: «Я разочарован таким обилием ограничений у существующих криптовалют, появится ли когда-нибудь совершенная криптовалюта?»
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [ninth episode](https://www.monerooutreach.org/breaking-monero/poisoned-outputs.php)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
...ть одна важная вещь, и одна абсолютно неважная. Сначала о неважном: кто-то спросил, чем мы увлекаемся, и я бы хотел рассказать о моём личном хобби. Мне нравятся деревья бонсай. Мне нравится выращивать их. Круто, когда пересаживаешь растение и обнаруживаешь, что корни имеют длину 5 футов, и понимаешь, что давно уже стоило пересадить его. Это забавно. Но есть действительно важная вещь, которую я хочу осуществить в июне. Это Monero Konferenco. 22 и 23 июня в Денвер, штат Колорадо, слетятся люди со всего мира, чтобы поговорить о Monero и обо всём, связанном с Monero. Приедут разработчики аппаратных средств, приедут люди из венчурных компаний и криптографы. Будет по крайней мере два исторических и потрясающих основы выступления, но пока я не могу о них рассказывать. Мне бы не хотелось поднимать лишний хайп, но вы ребята обязательно должны быть там, особенно учитывая, что билеты будут, то есть я хочу сказать, что сама конференция станет самой дешёвой в городе, то есть в принципе дешёвой. Если вы соберётесь посетить Consensus в Нью-Йорке, то вы потратите там безумное количество денег. Гораздо дешевле прилететь в Денвер и посетить эту конференцию, чем жить в Нью-Йорке и посещать Consensus.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [Monero Konferenco](https://monerokon.com/)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+
+_**Саранг:**_ Брендон, это чертовски заманчивое предложение, где можно получить больше информации?
+
+_**Шурэ:**_ Посетите monerokon.com.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [Monero Konferenco](https://monerokon.com/)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
...й из последних статей, написанных мной для Исследовательской лаборатории Monero, были критически проанализированы процесс цепной реакции, а также увеличение размера кольца, которое мы производим с течением времени. Наилучшим способом усовершенствования Monero является критика, поскольку мы сначала занимаем оборонительную и даже надменную позицию, а затем понимаем, что человек говорит дело, и нам необходимо что-то менять. Это будет положительный опыт, мы будем целую сессию обсуждать слабые стороны, позволяющие взломать Monero, и я надеюсь, что Джастин выступит модератором именно этой сессии. Посмотрим, как всё получится. Да, там ещё будет межконтинентальная «живая» панель, которую мы проведём совместно с участниками Zcon1, конференции, которая будет проходить в Хорватии. Фонд ZCash и сообщество Monero внесли свои пожертвования в Систему общественного краудфандинга, чтобы проведение Monero Konferenco стало возможным, а межконтинентальная панель будет посвящена вопросам правительственного регулирования и анонимности. В панели примут участие ребята из CoinCenter, Эрик Вурхис из ShapeShift, а также один из разработчиков ZCash. Мы отлично проведём время. Всё, я прекращаю болтать об этом.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/11_mailbag_episode_ru.md]: - [Zcon1](https://www.zfnd.org/zcon/)
- [Community Crowdfunding System](https://ccs.getmonero.org/)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/12_in_output_metadata_ru.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
...ктябре 2018 их создано было совсем немного, и выглядит так, как будто в течение этого периода кто-то пытался протестировать, как это работает. Примерно в это время состоялся форк Monero, и это интересно, если вы хотите исследовать блокчейн Monero, чтобы узнать, что люди делали в этот временной период, и эти транзакции с одним выходом определённо некоторым образом помогут в этом. Итак, это одна из вещей, которая показалась мне интересной во всём собранном «Исследовательской лабораторией абсурда», и что касалось распределения входов и выходов Monero. Это может быть очень полезно с точки зрения получения базового понимания того, делаете вы то же, что и большинство участников сети Monero, или нет. Ситуация может поменяться со временем, например, сейчас вы не можете проводить транзакции более чем с 16 выходами, поэтому количество транзакций более чем с 16 выходами уменьшается, но это хороший способ узнать, как люди в целом тратят свои средства. Саранг, не мог бы ты чуть подробнее рассказать, что на самом деле эти метаданные означают для людей, как к ним нужно относиться при проведении транзакций либо с нормальным количеством входов и выходов, либо с необычным количеством входов/выходов?
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/12_in_output_metadata_ru.md]: - [Excel document](https://github.com/noncesense-research-lab/tx_in_out_distribution)
- [Noncesense Research Lab](https://github.com/noncesense-research-lab/)
- [their github](https://github.com/noncesense-research-lab/tx_in_out_distribution)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/13_blockchain_explorer_ru.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
...к ExploreMonero.com, при помощи которых вы с лёгкостью можете верифицировать свои транзакции. То есть вы можете сказать: «Я отправил Monero», - и ввести сюда подробную информацию, и это действительно удобно и хорошо с этой точки зрения, но просто нужно понять, что если вы говорите о том, что получили Monero, и даёте подробную информацию о таком получении, то этот веб-сайт будут иметь очень сильные основания, чтобы подозревать вашу связь с транзакцией, по которой вами был сделан запрос. Он сможет связать данные с вашим IP и с тем IP-адресом, с которого вы будете делать запрос несколько раз, допустим, о пяти транзакциях, и высока вероятность, что ExploreMonero предположит, что вы связаны со всеми этими пятью транзакциями. Это просто напоминание о том, что когда вы используете блокчейн-эксплореры, вы раскрываете большой объём информации, и многие люди, в спешке пытающиеся получить информацию о своих транзакциях, часто забывают об этом в процессе. Я передаю слово Сарангу, который расскажет, что делать, если вы хотите получить информацию из блокчейн-эксплорера, но не хотите раскрывать свою информацию, или как использовать блокчейн-эксплорер, чтобы лучше защитить свои сетевые метаданные.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/13_blockchain_explorer_ru.md]: - [remote nodes in episode 7](https://www.monerooutreach.org/breaking-monero/remote-nodes.php)
- [xmrchain.net](https://xmrchain.net/)
- [ExploreMonero.com](https://www.exploremonero.com/)

File: monero-outreach-docs/translations/ru/transcriptions/breaking_monero/13_blockchain_explorer_ru.md [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
...ет раскрыта получающему серверу, уменьшится. Наилучшим решением, как и в случае с множеством других вещей, было бы создание своего локального узла. Если у вас есть свой локальный узел, у вас имеется полная синхронизированная копия блокчейна, вы получаете информацию о новых блоках и транзакциях по мере их поступления, и вы можете запрашивать эти данные локально. Если вы доверяете собственному узлу и собственному компьютеру, что, как мы надеемся, так и есть, то всё у вас будет хорошо. Такая информация не будет раскрыта. Также полезно знать, что даже если вы не хотите использовать решения с поддержкой командной строки или определённые локальные кошельки, которые работают вместе с вашим собственным узлом, полезно знать исходный код этих блокчейн-эксплореров, который обычно выложен в свободный доступ. Таким образом, вы всегда сможете запустить свой собственный локальный сетевой блокчейн-эксплорер, который будет подсоединяться к вашему узлу и запрашивать у него информацию. Если вы предпочитаете определённый вид эксплорера и имеете доступ к исходному коду, вы можете запустить собственный узел и иметь те же функции, к которым привыкли, но информация при этом не будет утекать куда-либо ещё.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/transcriptions/breaking_monero/13_blockchain_explorer_ru.md]: - [Tor](https://www.torproject.org/)

**Comment by v1docq47** [comment]
@lh1008 thank you! I added all links ;)

**Pull request merged**